packages/api/test/unit/beacon/testData/beacon.ts

```diff
@@ -30,11 +30,11 @@ export const testData: GenericServerTestCases<Api> = {
   // block
 
   getBlock: {
-    args: ["head"],
+    args: ["head", "json"],
```

Comment:
these test data are keyed by api name so we can only test with either "json" or "ssz". This is "json" because by default we use it, and majority of consumers use "json" so I'd go with it to maintain the test case going forward

Reply:
It doesn't look like we unit test the ssz format for state either.  I will be happy to add a unit test case after this gets merged.  Will be good practice for me.  I haven't worked with the API code much

https://github.com/ChainSafe/lodestar/blob/97dfcfc45c098e4a1363bb0213695916eeb94487/packages/api/test/unit/beacon/testData/debug.ts#L47-L54

Reply:
This test is just testing the shape/types of the input and output data, not any behavior. Since we added a param to the getBlock functions, we need to add a param here.